doc/changelog.rst

```diff
@@ -1,6 +1,7 @@
 Changelog
 =========
 
+* :feature:`-` Use ``-X` to set off ``config.root.run.stop_on_error`` (opposite to ``-x``)
```

Comment:
"`-X` can now be used to turn off stop-on-error behavior. Useful if you have it on by default through a configuration file"